package.json

```diff
@@ -73,6 +73,7 @@
     "prettier": "2.5.1",
     "redis": "3.1.2",
     "redis-lock": "0.1.4",
+    "regenerator-runtime": "^0.13.9",
```

Comment:
```suggestion
    "regenerator-runtime": "0.13.9",
```

Reply:
I'll let renovate do it, because it's going to need a lockfile update